tools/scripts/print-commits.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Minor pedantic remark: 
I generally prefer  to use `#!/usr/bin/env bash` because it searches the path for the version of the tool, bash in this case, installed on a POSIX/POSIX like environment.